src/components/Markdown/Headings.tsx

```diff
@@ -41,16 +39,18 @@ export const Headings: FunctionComponent<HeadingResolverProps> = ({
 
   const id = dataElement?.dataset.headingId ?? sanitize(title);
 
-  const isH3OrLarger = level < 4;
-
   return (
     <>
       <Heading
         as={elem}
-        color="blue.800"
+        backgroundColor={level >= 4 ? "gray.50" : undefined}
```

Comment:
I have mixed feelings about applying a background color to a heading element, but regardless this does make it difficult to visually distinguish Initializers from `props`

new
<img width="1062" alt="Screen Shot 2021-11-09 at 1 38 38 PM" src="https://user-images.githubusercontent.com/8749859/141011035-dfc2ab17-9bec-4525-b6e6-af79e08af9bf.png">

old
<img width="921" alt="Screen Shot 2021-11-09 at 1 38 33 PM" src="https://user-images.githubusercontent.com/8749859/141011033-810eafd3-5f88-4b75-84ea-a150acbdbb5c.png">